lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2226,9 +2226,6 @@ CompilerType TypeSystemSwiftTypeRef::GetChildCompilerTypeAtIndex(
           if (llvm::StringRef(AsMangledName(type))
                   .endswith("sSo18NSNotificationNameaD"))
             return GetTypeFromMangledTypename(ConstString("$sSo8NSStringCD"));
-          // FIXME: Private discriminators come out in a different format.
```

Comment:
How was this resolved in this patch?

Reply:
It's not entirely resolved, more changes required, one at least is to support validation. But it's required to support `Foundation.IndexPath.Storage` whose type would be ignored by this condition.

Reply:
See here: https://github.com/apple/llvm-project/pull/2330#discussion_r558426950